src/main/resources/test_files/runtime-spark/DataFrames/GroupbyClauseError2.iq

```diff
@@ -0,0 +1,6 @@
+(:JIQS: ShouldCrash; ErrorCode="XPDY0130" :)
+for $i in json-file("./src/main/resources/queries/conf-ex.json", 10)
+group by $target := $i.target
+return $target
+
+(: variable out of scope :)
```

Comment:
This test case throws a variable not found exception shown by XPDY0130. The variable defined within the scope of group by is not accessible at the return statement. This seems correct to me but I'd appreciate a sanity check.

Reply:
This query should return successfully. Group by variables are accessible all the way to and including the return clause.

Reply:
Correct behavior implemented